src/App.js

```diff
@@ -1,29 +1,27 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import LandingPage from './components/LandingPage';
+import Reservations from './components/Reservations';
+import MyReservations from './components/MyReservations';
+import DeleteReservations from './components/DeleteReservations';
+import HomePage from './components/HomePage';
+import CourseDetails from './components/CourseDetails';
+import ReservationForm from './components/ReservationForm';
+import Authentication from './components/Authentication';
+import AddCourse from './components/AddCourse';
 
+const App = () => (
```

Comment:
- [ ] Kindly, remove the old commented code found in:

```
Dashboard.jsx
HomePage.jsx
```

That way, the repo's cleanliness will be improved. 👍